esmvaltool/diag_scripts/thermodyn_diagtool/mkthe.py

```diff
@@ -0,0 +1,366 @@
+"""AUXILIARY FIELDS RETRIEVAL.
+
+Module for computation of the auxiliary variables needed by the tool.
+
+Here the thermodynamic diagnostic tool script computes
+some auxiliary variables.
+
+It computes equivalent potential temperatures and temperatures representative
+of the sensible and latent heat exchanges in the lower layers of the
+troposphere. Estimates of the boundary layer height and lifting condensation
+level are also provided.
+
+It ingests monthly mean fields of:
+- specific humidity (near-surface or 3D) (hus);
+- skin temperature (ts);
+- surface pressure (ps);
+- near-surface horizontal velocity (uas and vas);
+- surface turbulent sensible heat fluxes (hfss);
+- emission temperature (te).
+
+Authors: Frank Lunkeit and Valerio Lembo (University of Hamburg)
+
+Created on Fri Jun 15 10:06:30 2018
+"""
+import os
+from shutil import move
+from netCDF4 import Dataset
+import numpy as np
+from cdo import Cdo
+from esmvaltool.diag_scripts.thermodyn_diagtool import fourier_coefficients
+
+ALV = 2.5008e6    # Latent heat of vaporization
+G_0 = 9.81        # Gravity acceleration
+P_0 = 100000.     # reference pressure
+RV = 461.51       # Gas constant for water vapour
+T_MELT = 273.15   # freezing temp.
+AKAP = 0.286      # Kappa (Poisson constant R/Cp)
+GAS_CON = 287.0   # Gas constant
+RA_1 = 610.78     # Parameter for Magnus-Teten-Formula
+H_S = 300.        # stable boundary layer height (m)
+H_U = 1000.       # unstable boundary layer height (m)
+RIC_RS = 0.39     # Critical Richardson number for stable layer
+RIC_RU = 0.28     # Critical Richardson number for unstable layer
+L_C = 2501000 		        # latent heat of condensation
+SIGMAINV = 17636684.3034 	# inverse of the Stefan-Boltzmann constant
+
+
+# pylint: disable-msg=R0914
+# pylint: disable-msg=R0915
+# flake8: noqa
```

Comment:
Please do  not disable our quality checking tools, as they prevent use from automatically checking what the code quality is. There is no need to reach the perfect score for the pull request to be merged, so there is also no point in disabling the checks.
```suggestion
```

Reply:
I removed all the warning suppressions.